index.html

```diff
@@ -609,16 +610,19 @@ <h2 class="primary-txt">Contact me</h2>
                   />
                 </li>
                 <li>
-                  <label for="mail" hidden>Email</label><input
+                  <label for="email" hidden>Email</label>
+                  <input
                     type="email"
-                    placeholder="Write your email here"
-                    name="user_email"
-                    id="mail"
+                    placeholder="Write your email address here"
+                    name="userEmail"
+                    id="email"
                     required
                   />
+                  <div id="emailErrMsg" class="errorMsg"></div>
                 </li>
                 <li>
-                  <label for="msg" hidden>Message</label><textarea
+                  <label for="msg" hidden>Message</label>
+                  <textarea
                     placeholder="Write your message here"
                     name="user_message"
                     id="msg"
```

Comment:
### Your Error Message
![error-message (copy)](https://user-images.githubusercontent.com/93039359/181803057-626535c9-3ccf-4d6c-9cbb-ea6916c2452e.png)

- [ ] Yellow box is showing the correct place for the error to show because it is a requirement that the error message should be shown near the submit button. :thinking: